eggdrop.conf

```diff
@@ -1158,17 +1158,21 @@ server add ssl.example.net +7000
 #set sasl-timeout 15
 
 # To request the account-notify feature via CAP, set this to 1
-#set account-notify 0
+set account-notify 1
 
 # To request the extended-join feature via CAP, set this to 1
-#set extended-join 0
+set extended-join 1
 
 # To request the invite-notify feature via CAP, set this to 1
 #set invite-notify 0
 
-# To request the message-tags feature via CAP, set this to 1. NOTE: Enabling
-# this feature may interfere with RAW binds in scripts.
-#set message-tags 0
+# To request the message-tags feature via CAP, set this to 1
+set message-tags 1
+
+# To request the account-tag feature via CAP, set this to 1
+# This should only be enabled if necessary for imperfect account tracking
+# see doc/ACCOUNT for details
```

Comment:
A user may have other reasons they want to enable this than account-tracking, we may want to soften this language